src/helpers/ExtensionHelpers.js

```diff
@@ -0,0 +1,96 @@
+const bip39 = require('bip39');
+import * as HDKey from 'hdkey';
+import Toast from './responseHandler';
+import { toChecksumAddress, isAddress } from './addressUtils';
+
+const getAccounts = callback => {
+  const chrome = window.chrome;
+  chrome.storage.sync.get(null, callback);
+};
+
+const getPrivFromMnemonicWallet = (mnem, path) => {
+  const seed = bip39.mnemonicToSeedSync(mnem);
+  const hdKey = HDKey.fromMasterSeed(seed);
+  return hdKey.derive(path).privateKey;
+};
```

Comment:
you can use `import MnemonicWallet from '@/wallets'`, it will probably be better since it opens up other functionalities as well